tests/import-object.test.ts

```diff
@@ -22,6 +36,11 @@ function print(typ: Type, arg: any): any {
   return arg;
 }
 
+const memory = new WebAssembly.Memory({ initial: 2000, maximum: 2000 });
```

Comment:
@student-of @SasyaReddi note the need to create the memory in tests and in webstart so the printing functions can close over it. May need to think about the right place to do this. Also @lsteffkojr talked about aggregating some of the shared import stuff in a library, relevant here (and something similar needed for allocated bignums).

Reply:
I just created a new pull request for this idea of aggregating the import stuff.